mobile/src/screens/chat-list/chat-list.tsx

```diff
@@ -35,6 +37,8 @@ const ChatList: React.FC = () => {
   const dispatch = useAppDispatch();
   const navigation =
     useNavigation<NativeStackNavigationProp<ChatNavigationParameterList>>();
+  const [isModalVisible, setIsModalVisible] = useState<boolean>(false);
+  const [chatId, setChatId] = useState<number>();
```

Comment:
we should define what is this id for, because it's not obvious. Initial state for items that we get later should be null
```suggestion
  const [chatIdToDelete, setChatIdToDelete] = useState<number | null>(null);
```